Header header
uint32 software_version
uint8 software_subversion
uint32 hardware_version
